CSharp/Library/Microsoft.Bot.Connector.Shared/IMessageActivity.cs

```diff
@@ -58,14 +60,23 @@ public interface IMessageActivity : IActivity
         SuggestedActions SuggestedActions { get; set; }
 
         /// <summary>
-        /// Collection of Entity objects, each of which contains metadata about this activity. Each Entity object is typed.
+        /// Importance of the activity 
+        /// Valid values are "low", "normal", and "high". Default value is "normal."
         /// </summary>
-        IList<Entity> Entities { get; set; }
```

Comment:
This seems to break the Microsoft.Bot.Connector.Teams 'GetTextWithoutMentions' method.
https://github.com/OfficeDev/BotBuilder-MicrosoftTeams/issues/91